samples/drivers/crypto/src/main.c

```diff
@@ -17,9 +17,7 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(main);
 
-#ifdef CONFIG_CRYPTO_TINYCRYPT_SHIM
-#define CRYPTO_DRV_NAME CONFIG_CRYPTO_TINYCRYPT_SHIM_DRV_NAME
-#elif CONFIG_CRYPTO_MBEDTLS_SHIM
+#ifdef CONFIG_CRYPTO_MBEDTLS_SHIM
```

Comment:
Adding a comment here because I cannot do it on the proper line.
There are still comments referring to TinyCrypt in this file (at lines 8, 266, 419, 531). Can you please also remove them?